code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
```

Comment:
I think I rather use /me for emotes. Key words like /say help put out that it's a thing you are doing. Or maybe an /emote for them. It helps with less confusing options